### ISSUE TITLE
Win+D combination not draws objects if set "2D" + "expand" window strech mode in project settings

### ISSUE THREAD

**Issue by RikKargones** [godotengine/godot#52283]
### Godot version

3.3.3.stable

### System information

GLES 2.0 project

### Issue description

Issue in topic's name.

In my project Camera2D show nothing on the screen, then I press Win+D combination and then expand window back.

It works in Debug and Realise builds and in Godot 3.3 it works at the same way.

However, if you have separate CanvasLayer, all nodes under it will be showed. Further research show (and I find it accidentally) what actions breaks somithing inside Camera2D (more likely) or root CanvasLayer, becouse $Camera2D.get_camera_screen_center() returns vector with "-1.#IND" in y parameter.

### Steps to reproduce

Here project what represents this issue.

### Minimal reproduction project

[New.zip](https://github.com/godotengine/godot/files/7084773/New.zip)

**Comment by Calinou** [comment]
Can you get something to display if you resize the window after restoring it?

**Comment by RikKargones** [comment]
No. It's stay the same after resizing, set fullscreen or ever changing def screen resolution and strech mode trough code.
I'm trying to inspect Camera2D code in engine, but I have too little knowlege about how it works.

**Comment by Calinou** [comment]
`-1.#IND` is likely `-INF` (`-Math_INF` in C++), also known as negative floating-point infinity.
